src/Workspaces/CSharp/Portable/Rename/LocalConflictVisitor.cs

```diff
@@ -38,11 +38,16 @@ public override void VisitMethodDeclaration(MethodDeclarationSyntax node)
 
         public override void VisitBlock(BlockSyntax node)
         {
+            VisitBlockStatements(node, node.Statements);
```

Comment:
Do you mean to create generic method like:
`private void VisitBlockStatements<T>(SyntaxNode node, T statements) where T : IEnumerable<SyntaxNode>`